### PR TITLE
[AIC-PY] cli-mate demo

### PR DESCRIPTION
**Pull request by jonathanlastmileai**
[AIC-PY] cli-mate demo

---
Stack created with [Sapling](https://sapling-scm.com). Best reviewed with [ReviewStack](https://reviewstack.dev/lastmile-ai/aiconfig/pull/224).
* __->__ #224
* #240

### CODE REVIEW

File: python/demo/cli-mate.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
+import asyncio
```
Comment (saqadri) [inline python/demo/cli-mate.py:1]: Move to cookbooks

File: python/demo/cli-mate.py [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    i = 0
+    while True:
+        try:
+            user_input = input("\nQuery: ")
+            if user_input.strip() == "":
+                continue
+            if user_input.strip().startswith("reload") or i == 0:
+                user_input = deprefix(user_input.strip(), "reload")
+                source_code = open(source_code_file, "r").read()
+                prompt = (
```
Comment (saqadri) [inline python/demo/cli-mate.py:91]: I think this should be in the aiconfig itself.

See the prompt routing cookbook to see how to route to the right prompt using aiconfig

File: cli-mate.aiconfig.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (saqadri) [inline cli-mate.aiconfig.json:1]: Move to cookbooks

**Review by saqadri — APPROVED** [review]
Love the name. Please move to a cookbook with a readme guide. Overall a good example to showcase

File: python/demo/cli-mate.py (hunk 2 above)
Comment (jonathanlastmileai) [reply python/demo/cli-mate.py]: I'm not sure how to do this in a loop - every prompt in this script is added dynamically. Any advice? cc @LastMileNewGrad

**Pull request merged**